### DISCUSSION TITLE
Replicate “Minute-scale oscillatory sequences in medial entorhinal cortex” in other dandisets

### DISCUSSION THREAD

**Discussion by stephprince** [dandi/helpdesk#156] [Data reanalysis ideas]
## Motivation
[Gonzalo Cogno et al. (2024)](https://doi.org/10.1038/s41586-023-06864-1) found that neural activity in the mouse medial entorhinal cortex (MEC) can be organized into ultraslow oscillations, with periods ranging from tens of seconds to minutes. During these slow oscillations, neural activity was also organized into periodic sequences. These sequences were not related to location, running direction, or scheduled rewards, and they were specific to MEC. 

This finding is a good candidate for a data reanalysis study because the claim is independent of specific behavioral/task variables and was observed in both calcium imaging and electrophysiological data. The authors also shared [their data](https://search.kg.ebrains.eu/instances/a25f722e-f9b1-4f32-80bc-37206c2d1536) and [their code](https://github.com/soledadgcogno/Ultraslow-oscillatory-sequences)! 

There are a few dandisets with recordings from [entorhinal cortex](https://dandiarchive.org/dandiset/search?search=entorhinal%20cortex), and it would be interesting to replicate these findings in these datasets; incorporating additional electrophysiological recordings, applying different data preprocessing methods, using data from other species, and using data recorded under different experimental conditions. 

## Work performed so far
As part of NeuroDataReHack 2024, @rly, @bendichter, and I made some progress on an initial data reuse demonstration. Code can be found [here](https://github.com/rly/replicate-gonzalo-cogno-2023).

1. We replicated part of their findings (Figure 1) using their shared data and our own Python code: ([figure1_autocorr.ipynb](https://github.com/rly/replicate-gonzalo-cogno-2023/blob/main/figure1_autocorr.ipynb), and [figure2_2pautocorr.ipynb](https://github.com/rly/replicate-gonzalo-cogno-2023/blob/main/figure1_2pautocorr.ipynb))
2. We replicated part of their findings using one of the shared datasets on DANDI ([dandiset 000053](https://dandiarchive.org/dandiset/000053?search=entorhinal%20cortex&pos=6)) by [Mallory et al., 2021](https://www.nature.com/articles/s41467-021-20936-8): ([figure1_newdata.ipynb](https://github.com/rly/replicate-gonzalo-cogno-2023/blob/main/figure1_newdata.ipynb))

## Next steps
1. Continue to replicate the other findings in the paper. We developed some code to replicate their finding of minute-scale oscillations at a single cell level, but it would be great to replicate their findings of oscillatory sequences at a population level.
2. Contact the authors, convert their data to NWB, and upload it to DANDI. The shared neuropixels recordings were saved in NWB format, but the calcium imaging data and some additional metadata was shared in other formats. It would be great to convert their full dataset to NWB to improve the interoperability of any analysis code across datasets and for future reuse. 
3. Leverage the author’s data combined with other dandisets to expand upon the original findings. The authors mention in the discussion that their work brings up further questions about how these sequences present across a variety of behavioral conditions and how they interface with the dynamics of grid cells and head direction cells. A data reanalysis project could be a great opportunity to pursue questions such as:
    - Is this type of oscillatory activity also present during navigation tasks?
    - How do these slow oscillatory cells relate to grid cells? Are they grid cells or are they not?
    - Is this phenomenon present in other species?
    - Does this oscillatory phenomenon relate to behavior or task accuracy?

## If you’re interested
We would like to propose this as a potential project for collaboration and/or hand it off to a new researcher to continue to develop. We could also contact the authors of the initial paper and dandiset 000053 to see if they would like to be involved.

**Comment by bendichter** [comment]
Following up on next steps 1 and 3, and building on the NeuroDataReHack work in [rly/replicate-gonzalo-cogno-2023](https://github.com/rly/replicate-gonzalo-cogno-2023). We put together a fuller attempt at the population-level replication and extended it to the navigation and cross-species questions. Code and a written walkthrough are here: https://github.com/catalystneuro/mec-ultraslow-replication

A few things worth reporting.

**The population sequence test needs a positive control, and single-cell rhythmicity is not it.** The single-cell ultraslow rhythmicity from Figure 1 is easy to reproduce, but it turns out to be non-specific: 50 to 75 percent of cells look ultraslow and periodic in essentially every dataset we ran, including visual cortex and free foraging. The paper's own Figure 5 says the same thing (parasubiculum and visual cortex cells are "ultraslow and periodic" but form no sequences). So the load-bearing test is the population sequence test, not the single-cell spectrum. We built the pipeline around the original Neuropixels wheel data as a positive control, on the principle that a detector that cannot fire where the effect is known to be present cannot be trusted to report a null anywhere else. Two subtle statistical traps (a band-power statistic that is blind by construction, and a permutation resolution floor that makes FDR unreachable) only surfaced because that control existed.

**On the navigation question: we do not detect the oscillatory sequences in mouse MEC during navigation.** Across 134 mouse MEC navigation sessions (0/20 in the 000053 VR track, 3/114 in an X-maze dataset) the sequence rate is at chance, and a within-session visual-cortex control is actually higher than MEC, which is the opposite of an intrinsic MEC-specific effect. This is consistent with the authors' own speculation that the sequences might reset in the presence of strong landmarks or sensory feedback. It is a partial answer to their open question, not proof that darkness is required: the paper never ran another condition, and a null cannot separate "sensory drive abolishes the sequences" from "insufficient statistical power".

**On the cross-species question: the one apparent positive is a confound.** Macaque entorhinal cortex (000897) looked like a striking replication at first (8/15 sessions), but the population signal tracks trial-onset density over the multi-hour session rather than an intrinsic rhythm, so it is task-engagement block structure. The mouse visual-cortex "positive" is the lap cycle. Both are documented with dedicated controls in the repo.

**One concrete point that supports next-step 2.** The EBRAINS wheel recording is the only public data in the exact condition where the effect is known to exist, and it is what makes a null interpretable. Getting it onto DANDI in NWB would be genuinely valuable for anyone continuing this.

Happy to hand this off or keep going. The grid-cell-identity question in the list is the natural next one, and we have not touched it yet.
